creates python package to validate schemas

pablopalacios opened the pull request:
Signed-off-by: Pablo Palácios <ppalacios992@gmail.com>

pablopalacios committed:
creates python package to validate schemas

Signed-off-by: Pablo Palácios <ppalacios992@gmail.com>

otavio reviewed (changes requested):
Besides the CI error, one thing which concerns me is the path name used. I would prefer it uses just python or uses the complete path name.

pablopalacios commented:
I've still keeped the old bash tests since it was easier than translate it to Python.

pablopalacios committed:
exceptions: creates jsonschema.ValidationError wrapper

With this modification, users doesn't have to test against jsonschema
exceptions.

Signed-off-by: Pablo Palácios <ppalacios992@gmail.com>
